packages/config-plugins/src/ios/__tests__/ProvisioningProfile-test.ts

```diff
@@ -13,6 +13,45 @@ describe('ProvisioningProfile module', () => {
     const projectRoot = '/testproject';
     const pbxProjPath = 'ios/testproject.xcodeproj/project.pbxproj';
 
+    afterEach(() => {
+      vol.reset();
+    });
+
+    it('configures the project.pbxproj file for multitarget projects', () => {
+      vol.fromJSON(
+        {
+          [pbxProjPath]: originalFs.readFileSync(
+            path.join(__dirname, 'fixtures/project-multitarget.pbxproj'),
+            'utf-8'
+          ),
+        },
+        projectRoot
+      );
+      setProvisioningProfileForPbxproj(projectRoot, {
+        profileName: '*[expo] com.swmansion.dominik.abcd.v2 AppStore 2020-07-24T07:56:22.983Z',
+        appleTeamId: 'J5FM626PE2',
+      });
+      const pbxprojContents = fs.readFileSync(path.join(projectRoot, pbxProjPath), 'utf-8');
+      expect(pbxprojContents).toMatchSnapshot();
+    });
+    it('configures the project.pbxproj file for multitarget projects without existing target attributes', () => {
+      vol.fromJSON(
+        {
+          [pbxProjPath]: originalFs.readFileSync(
+            path.join(__dirname, 'fixtures/project-multitarget-missing-targetattributes.pbxproj'),
+            'utf-8'
+          ),
+        },
+        projectRoot
+      );
+      setProvisioningProfileForPbxproj(projectRoot, {
+        profileName: '*[expo] com.swmansion.dominik.abcd.v2 AppStore 2020-07-24T07:56:22.983Z',
```

Comment:
the same as above, you are testing correctly your changes but still, you are treating that as single target project